meson.build

```diff
@@ -2692,7 +2692,7 @@ foreach test: tests
     endif
 
     test(t['name'] / 'pg_regress',
-      sh,
```

Comment:
I think this will have removed the last 'sh' user, right? If so, let's remove it.

Reply:
Yes, it was last. Removing and merging.